ubuntudesign/developer.ubuntu.com
# Add missing patterns and utilities

## Done

Add card and divider patterns
Add margin and padding collapse utilities

## QA

Run ./run to get started, then go to http://localhost:8015/community-resources
Make sure the files exist and that they are being pulled into global.scss
I’ve added the collapse classes to the news feed list in templates/includes/components/feed_cards.html so if you remove them you can see that it works
